modules/apps/portal-security/portal-security-ldap-impl/src/main/java/com/liferay/portal/security/ldap/internal/exportimport/LDAPUserExporterImpl.java

```diff
@@ -97,7 +97,9 @@ public void exportUser(
 			contact.getContactId());
 
 		if (user.isDefaultUser() ||
-			(user.getStatus() != WorkflowConstants.STATUS_APPROVED)) {
+			((user.getStatus() != WorkflowConstants.STATUS_APPROVED) &&
+			 (user.getStatus() != WorkflowConstants.STATUS_INACTIVE)) ||
+			_isAnonymousUser(user)) {
```

Comment:
Hey @istvansajtos 
I read your comment:
> My changes intend to enable exporting INACTIVE users to LDAP with the exception of anonymous users.

I think the condition can be simplified as such, what do you think:
```
if (user.isDefaultUser() ||
	(user.getStatus() != WorkflowConstants.STATUS_APPROVED) ||
	_isAnonymousUser(user)) {
}

```

Reply:
For example: an inactive user who is not defaultUser and is not anonymous, then he shall pass all 3 checks.

Reply:
Hey @arthurchan35 ,

Thinking trough the modification, I don't think that would give the expected result. For example: Admin deactivated User1 who has INACTIVE status now. 

The updated condition does the following checks:
- _user.isDefaultUser()_ => **false**
- _(user.getStatus() != WorkflowConstants.STATUS_APPROVED)_  => **true**, export won't happen

While for the original condition:
- user.isDefaultUser() => **false**
- ((user.getStatus() != WorkflowConstants.STATUS_APPROVED) &&
(user.getStatus() != WorkflowConstants.STATUS_INACTIVE)) => **false**, while for STATUS_INCOMPLETE or STATUS_PENDING it will give **true**
- _isAnonymousUser(user) => **false**

What do you think?

Reply:
@istvansajtos
You are correct, I didnt think it straight, dropped my commit!